packages/db/prisma/schema.prisma

```diff
@@ -64,3 +89,136 @@ model VerificationToken {
 
     @@unique([identifier, token])
 }
+
+// ======================================
+// End Mostly Next-Auth.js
+// ======================================
+
+// ======================================
+// Start Enums
+// ======================================
+
+enum Language {
+    EN
+    DE
+    IT
+    FR
+    ES
+}
+
+enum EventCalendarType {
+    BLOCKER
+}
+
+enum EventResponseStatus {
+    ACCEPTED
+    DECLINED
+    NEEDS_ACTION
+    TENTATIVE
+}
+
+enum Membership {
+    FREE
+    TRIAL
+    PRO
+    PREMIUM
+}
+
+// ======================================
+// End Enums
+// ======================================
+
+// ======================================
+// Start AgreeTo Specifics
+// ======================================
+
+model Preference {
+    id             String   @id @default(cuid())
+    userId         String   @unique
+    formatLanguage Language @default(EN)
+
+    createdAt DateTime @default(now())
+    updatedAt DateTime @default(now()) @updatedAt
+
+    user User @relation(fields: [userId], references: [id])
+}
+
+model EventGroup {
+    id              String  @id @default(cuid())
+    userId          String
+    accountId       String
+    title           String
+    appointmentUrl  String?
+    events          Event[]
+    createBlocker   Boolean @default(false)
+    isSelectionDone Boolean @default(false)
+
+    deletedAt DateTime?
+    createdAt DateTime  @default(now())
+    updatedAt DateTime  @default(now()) @updatedAt
+
+    account Account @relation(fields: [accountId], references: [id])
+}
+
+model Event {
```

Comment:
I'm not very familiar with this model but wonder if you see any differences to [cal.com's event model](https://github.dev/calcom/cal.com/blob/0cb278d1098dc7405849c7328767fdefbf149213/packages/prisma/schema.prisma#L32-L33) that surprised you, @juliusmarminge?